SPECS/systemd/systemd.spec

```diff
@@ -1,62 +1,342 @@
-%define majver %(echo %{version} | cut -d. -f 1)
-Summary:        Systemd
+
+# Temporary until azure-release-common is updated to carry macros.dist that defines %azure
+%global azure 3
+
+# We ship a .pc file but don't want to have a dep on pkg-config. We
+# strip the automatically generated dep here and instead co-own the
+# directory.
+%global __requires_exclude pkg-config
+
+%global pkgdir %{_prefix}/lib/systemd
+%global system_unit_dir %{pkgdir}/system
+%global user_unit_dir %{pkgdir}/user
+
+%if 0%{?__isa_bits} == 64
+%global elf_bits (64bit)
+%global elf_suffix ()%{elf_bits}
+%endif
+
+%bcond bzip2  1
+%bcond gnutls 0
+%bcond lz4    1
+%bcond xz     1
+%bcond zlib   1
+%bcond zstd   1
+
+# Bootstrap may be needed to break circular dependencies with cryptsetup,
+# e.g. when re-building cryptsetup on a json-c SONAME-bump.
+%bcond bootstrap 0
+%bcond tests     1
+%bcond lto       1
+
+# Support for quick builds with rpmbuild --build-in-place.
+# See README.build-in-place.
```

Comment:
readme was removed but i can add it back, though i am pretty sure our setup will not actually work with build-in-place without significant other changes. I left this in place to reduce delta with the fedora spec; i am assuming nobody reading this spec file wants to attempt build-in-place and so will likely just ignore this section, but it can be removed (at least for now, as i would very much like build-in-place working eventually). I didn't see its removal as important.

Reply:
I'd suggest removing this particular macro completely to just make the spec a bit easier to read, since, like you noticed, we may not have much use of it in predictable future and it doesn't ease enabling/disabling specific features like the macros set with `%bcond`s above. Not a huge concern for me right now, though.